.github/dependabot.yml

```
@@ -8,8 +8,8 @@ version: 2
 updates:
   # Maintain dependencies in our GitHub Workflows
   - package-ecosystem: github-actions
-    directory: "/"
+    directory: /
     schedule:
```

I've raised this change suggestion for discussion in https://github.com/jupyterhub/team-compass/issues/636 as well, please opine in that issue about it.

```suggestion
    labels: [ci]
    schedule:
```